src/ray/gcs/gcs_server/gcs_worker_manager.cc

```diff
@@ -52,8 +52,15 @@ void GcsWorkerManager::HandleReportWorkerFailure(
                      << ", address = " << worker_address.ip_address();
     } else {
       stats::UnintentionalWorkerFailures.Record(1);
+      // Only publish worker_id and raylet_id in address as they are the only fields used
```

Comment:
How about modify `AsyncSubscribeToWorkerFailures` synchronously, only showing `worker_id` and `raylet_id`?